examples/sycl/pvc/flash_attention_v2/CMakeLists.txt

```diff
@@ -28,11 +28,20 @@
 
 
 cutlass_example_add_executable(
-  pvc_flash_attention
-  pvc_flash_attn.cpp
+  pvc_flash_attention_64_causal
```

Comment:
Why 4 different binaries? Is the goal to allow parallel compilation? If so why not only make the kernel in different modules?

Reply:
I created 4 for better debugging purpose. Agreed it does not need to be in the final version.